### PR TITLE
Update Docs Develop Section for SDK inconsistency

### PR DESCRIPTION
**Pull request by 0xMenna01**

### CODE REVIEW

File: docs/develop/QuickReference.md [hunk 1]
```diff
@@ -736,7 +730,7 @@ Reactions are your signs to `Upvote` or `Downvote` a post.
 
 ```ts
   const postId = '1' // Post Id you want to react on.
-  const substrateApi = api.blockchain
+  const substrateApi = await api.blockchain.api
```
Comment (olehmell) [inline docs/develop/QuickReference.md:733]:
```suggestion
  const substrateApi = await api.substrateApi
```

File: docs/develop/how-to-guides/ConnectToSubsocial.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ You need to initialize the Subsocial SDK with the following syntax:
 
 ```typescript
 
-/* import newFlatSubsocialApi and util functions */
+/* import newSubsocialApi and util functions */
```
Comment (olehmell) [inline docs/develop/how-to-guides/ConnectToSubsocial.md:27]:
```suggestion
/* import SubsocialApi and util functions */
```

File: docs/develop/how-to-guides/profiles/CreateProfiles.md [hunk 3]
```diff
@@ -8,8 +8,10 @@ displayed_sidebar: developSidebar
 
 ## Create a profile
 
+To create a profile it is first necessary to create a space and then set it as the profile.
+
 ```
-substateApi.tx.profiles.createProfile(content)
+substrateApi.tx.spaces.createSpace(content, permissionsOpt)
```
Comment (olehmell) [inline docs/develop/how-to-guides/profiles/CreateProfiles.md:14]: I think need to add `profiles.setProfile` too

**Review by olehmell — CHANGES_REQUESTED** [review]

File: docs/develop/how-to-guides/profiles/CreateProfiles.md (hunk 3 above)
Comment (0xMenna01) [reply docs/develop/how-to-guides/profiles/CreateProfiles.md:14]: done thank you.

**Review by olehmell — APPROVED** [review]

**Pull request merged**